src/components/v5/frame/ColonyHome/partials/Agreements/Agreements.tsx

```diff
@@ -72,7 +71,8 @@ const Agreements = () => {
                 <span className="text-3">
                   {newestAgreement.decisionData?.title}
                 </span>
-                <MessageNumber message={1} />
+                {/* @TODO Re-enable once a Motion supports the comments feature */}
+                {/* <MessageNumber message={1} /> */}
```

Comment:
Normally, I'd say to remove this altogether since we can always restore this from `git` history